templates/archives/our_team.jinja2

```diff
@@ -232,33 +229,34 @@ About the Lab
              <div class="team-member">
                  <img class="mx-auto" src="{{ static("img/team/Xu_Katherine.jpg") }}" alt="Katherine Xu">
                  <h4>Katherine Xu</h4>
-                 <p class="text-muted">UROP<br>Gender/Novels<br>Comp/Hist</p>
+                 <p class="text-muted"></p>
              </div>
          </div>
 
         <div class="col-6 col-sm-4 col-md-2 text-center">
              <div class="team-member">
                  <img class="mx-auto" src="{{ static("img/team/Yen_Derek.jpg") }}" alt="Derek Yen">
                  <h4>Derek Yen</h4>
-                 <p class="text-muted">UROP<br>Gender/Novels<br>Comp/Hist</p>
+                 <p class="text-muted"></p>
              </div>
         </div>
 
         <div class="col-6 col-sm-4 col-md-2 text-center">
             <div class="team-member">
                  <img class="mx-auto" src="{{ static("img/team/Zhi_Sophia.jpg") }}" alt="Sophia Zhi">
                  <h4>Sophia Zhi</h4>
-                 <p class="text-muted">UROP<br>Gender/Novels<br>Comp/Hist</p>
+                 <p class="text-muted"></p>
             </div>
         </div>
 
     </div>
 
 
 
```

Comment:
Maybe for this project it would also be appropriate to include archivists and librarians who helped us with the archives and with scanning